.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2.0
+jobs:
+  build:
+    environment:
+      - BRANCH: "master"
+    steps:
+      - checkout:
+          path: ~/openwrt_packages
+
+      - run:
+          name: Check changes / verify commits
+          working_directory: ~/openwrt_packages
+          command: |
+             cat >> $BASH_ENV <<EOF
```

Comment:
I think, that we've discussed this already, nobody sane is going to suffer with YAML pox voluntarily :-) so it would be better to keep the YAML file as short as possible and do the rest in the external Bash/Python/Perl scripts instead (probably in separate directory `scripts/ci`) ? Ideally those checks could be run as make targets `make ci-formal-checks` (defined in `include/ci-commands.mk`) even localy by the developers, so they could simply check the contributions before sending patch/PR, debug/develop those CI checks etc. This way it's probably also CI agnostic, so in the end it doesn't matter if it's run on CircleCI or GitLab CI or whatever we choose in the future.

Reply:
Good point, I'll surely work on that